Resources/Prototypes/Catalog/Cargo/cargo_materials.yml

```
@@ -58,6 +58,16 @@
   category: Materials
   group: market
 
+- type: cargoProduct
+  id: MaterialCardboard
```

```suggestion
  id: CardboardMaterial
```

There's a naming issue here since there's MaterialCardboard in Materials/materials.yml and then again here, which causes a naming issue from the ftl file. Causes the stack to be named after the box.